docs/CHANGELOG.md

```
@@ -17,6 +20,8 @@ the old hash code.
 
 ### Closed issues
 
+#6258.
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)